.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Instead of committing this file, I would add .vscode to the .gitignore file (as I've done with .idea for PyCharm)

Reply:
Oops, I've just seen that the .idea directory it still in the repo! Need to fix that :see_no_evil: